champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/data/BlockTossObject.java

```diff
@@ -297,7 +308,7 @@ public void impact(Player caster) {
             UtilVelocity.velocity(vehicle, caster, velocityData);
         }
 
-        // Damage and heal
+        // Damage
         final List<KeyValue<LivingEntity, EntityProperty>> nearby = UtilEntity.getNearbyEntities(caster, impactLocation, radius, EntityProperty.ALL);
```

Comment:
Replace `EntityProperty.ALL` with `EntityProperty.ENEMY` if you removed the healing factor.